active/0000-better-in-repo-addons.md

```diff
@@ -0,0 +1,149 @@
+- Start Date: 2016-07-15
+- RFC PR:
+- ember-cli Issue:
+
+# Summary
+
+Make in-repo-addons as self-contained as full addons.
+
+# Motivation
+
+An in-repo addon is a great way to isolate code in an application without
+adding the extra costs associated with version management. In some cases,
+moving code from `app/` to `lib/my-addon/` is the first step towards
+releasing it as a full, open-source addon.
+
+Unfortunately, there are two important ways in which in-repo addons fail
+to meet the goal of isolation:
+
+ 1. They cannot declare NPM dependencies
```

Comment:
there is a bug, when resolving addons. They resolving code does not take into account the node resolution algorithm. That wont fix this in all cases, but it will help some :)